webpack.config.js

```diff
@@ -51,6 +55,20 @@ function getAliasesForExtensions() {
 	return aliasesMap;
 }
 
+/*
+ * Find package.json for file at `filepath`.
+ * Return `true` if it has a property whose key is `PROPKEY_ESNEXT` or 'module'.
+ * Taken from http://2ality.com/2017/06/pkg-esnext.html
+ */
+function hasPkgEsnext( filepath ) {
+	const pkgRoot = findRoot( filepath );
+	const packageJsonPath = path.resolve( pkgRoot, 'package.json' );
+	const packageJsonText = fs.readFileSync( packageJsonPath,
+		{ encoding: 'utf-8' } );
```

Comment:
fun fact! I recently discovered that we can pass the `options` value as a _string_, in which case it will be interpreted as the encoding…

```js
fs.readFileSync( packageJsonPath, 'utf-8' );
```

Reply:
Prefer that notation? I like the explicit `encoding` object key...